lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -181,6 +186,18 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
         deregisterFromKeyboardNotifications()
     }
 
+    @IBAction func addCoverButtonTapped(_ sender: UIButton) {
+        if AVCaptureDevice.authorizationStatus(for: .video) ==  .authorized {
+            present(cameraViewController, animated: true, completion: nil)
+        } else {
+            AVCaptureDevice.requestAccess(for: .video) { granted in
+                if granted {
+                    self.present(self.cameraViewController, animated: true, completion: nil)
```

Comment:
I think this needs to be on the main thread. I was getting a crash on this line in my testing:
```
Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'accessing _cachedSystemAnimationFence requires the main thread'
*** 
First throw call stack:
(0x18e896fe0 0x18d2f8538 0x18e896eb4 0x18f32f760 0x194c44fd0 0x194c9db0c 0x194a827a8 0x1949cf710 0x194a3da80 0x194a3da54 0x194ac26dc 0x194d38440 0x1a304ec80 0x194d39cd0 0x194d3c7e4 0x194ab9794 0x1a304e53c 0x10007ef4c 0x10006f984 0x190aa92f8 0x190aac3d4 0x18d98af30 0x18d98aea0 0x100ffda10 0x10100b990 0x10100d630 0x10100d39c 0x18d957100 0x18d956cac)
libc++abi.dylib: terminating with uncaught exception of type NSException
```